website/src/app/kb/quickstart/readme.mdx

```diff
@@ -80,8 +93,9 @@ Site Details page where the gateway will now be listed and shown as online.
 
 ### Add a Resource
 
-A Resource is anything the site-gateway can reach, that you'd like to give users
-access to. Examples include servers, databases, subnets, etc...
+A Resource is simply anything the gateway can reach that you'd like to give
+users access to. Examples include servers, databases, applications, and even
+entire subnets.
```

Comment:
I would suggest something like this: 
```suggestion
entire subnets.

The more granular resources are the better your control over the security policies around it.
```
CIDR ranges will be common but should not be encouraged.

Reply:
I'll this inside an `Alert color=info`